### PR TITLE
UCR API: Add Type Information to Report Columns

### PR DESCRIPTION
**Pull request by sheelio**
@NoahCarnahan 

I realized we didn't have type information (which makes it hard to identify if something is a expanded column).

When getting report data returned, I wasn't able to figure out an easy way to send the expanded choice.  For example, if I expand on gender, right now the data returned is like gender_0, gender_1 and has no info that 0 maps to male, 1 maps to female, etc. 

Should I create a separate ticket for that?

### CODE REVIEW

File: corehq/apps/api/tests.py [hunk 1]
```diff
@@ -1638,6 +1638,13 @@ def setUpClass(cls):
                 "type": "field",
                 "field": "my_field",
                 "aggregation": "simple",
+            },
+            {
+                "column_id": 'expand',
+                "display": "expand display",
+                "type": "expanded",
+                "field": "my_field",
+                "max_expansion": 10,
```
Comment (dimagimon) [inline corehq/apps/api/tests.py]: W291 trailing whitespace

File: corehq/apps/api/tests.py [hunk 2]
```diff
@@ -1691,7 +1698,7 @@ def test_get_detail(self):
         )
 
         self.assertEqual(
-            [{"column_id": c['column_id'], "display": c['display']} for c in self.report_columns],
+            [{"column_id": c['column_id'], "display": c['display'], "type": c['type']} for c in self.report_columns],
```
Comment (dimagimon) [inline corehq/apps/api/tests.py]: E501 line too long (117 > 115 characters)

**Comment by czue** [comment]
lgtm

**Comment by NoahCarnahan** [comment]
@sheelio Sure, you can create a separate ticket for that

**Pull request merged**